packages/addons/service/prometheus-node-exporter/package.mk

```diff
@@ -0,0 +1,46 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2009-2017 Lukas Rusak (lrusak@libreelec.tv)
+# Copyright (C) 2017-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="prometheus-node-exporter"
+PKG_VERSION="1.2.2"
+PKG_SHA256="3b7b710dad97d9d2b4cb8c3f166ee1c86f629cce59062b09d4fb22459163ec86"
+PKG_REV="101"
+PKG_ARCH="any"
+PKG_LICENSE="ASL"
+PKG_SITE="https://github.com/prometheus/node_exporter"
+PKG_URL="https://github.com/prometheus/node_exporter/archive/refs/tags/v${PKG_VERSION}.tar.gz"
+PKG_DEPENDS_TARGET="toolchain go:host systemd"
+PKG_SECTION="service"
+PKG_SHORTDESC="Exporter for machine metrics."
+PKG_LONGDESC="Prometheus exporter for hardware and OS metrics exposed by *NIX kernels, written in Go with pluggable metric collectors."
+PKG_TOOLCHAIN="manual"
+
+# https://github.com/prometheus/node_exporter/releases
+export PKG_GIT_COMMIT="26645363b486e12be40af7ce4fc91e731a33104e"
```

Comment:
is there a way we can reuse pkg_version ? nobody will remember to bump the additional value here :)

Reply:
This is only use for the revision part of this output

```
# prometheus-node-exporter --version
node_exporter, version 1.2.2 (branch: master, revision: 26645363b486e12be40af7ce4fc91e731a33104e)
  build user:       root@libreelec
  build date:       20210815-11:03:08
  go version:       go1.16.6
  platform:         linux/arm
  
```
So I change it to the LE addon package revision ( 100 actually) 

Ok for you ? :)

Reply:
yea good enough